react-carousel-component/src/index.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Carousel from './carousel';
+
+const pokemon = [
+  {
+    name: 'squirtle',
+    id: 1
+  },
+  {
+    name: 'sandshrew',
+    id: 2
+  },
+  {
+    name: 'umbreon',
+    id: 3
+  },
+  {
+    name: 'mew',
+    id: 4
+  },
+  {
+    name: 'magikarp',
+    id: 5
+  }
+];
```

Comment:
As mentioned in the previous comment, here instead of storing the `name` you should have an `image` or `url` property set to the file path of the image. You can then use an img tag in your jsx.